vendor-extra/ContentPageBundle/src/Controller/ContentController.php

```diff
@@ -62,7 +64,10 @@ function (ResponseInterface $response) use ($request) : Response {
                     return new Response(
                         $this->twig->render(
                             $this->template,
-                            $event->getContext() + ['title' => $event->getTitle(), 'content' => $event->getContent()]
+                            $event->getContext() + [
+                                'title' => $event->getTitle(),
+                                'content' => json_decode(json_encode($event->getContent()), true),
```

Comment:
The problem is that Twig's `with` _only_ works with arrays, not `ArrayAccess` objects. There's probably a far better way round it.

Reply:
Is this the problem line: https://github.com/twigphp/Twig/blob/2.x/src/Node/WithNode.php#L43

Could be replaced with `is_iterable` if twig targeted only PHP 7+ not sure if theres a simple fix

Reply:
Opened https://github.com/twigphp/Twig/pull/2901.

Reply:
Merged in, but only for `Traversable`. Will see if there's a way round it until that's released.

Reply:
Patching in 2e470a8853c8bde1cd9eef7fc3499b47b30dcf93.